backend/votes/serializer.py

```diff
@@ -0,0 +1,11 @@
+
+from rest_framework import serializers
+
+from unions.models import Union, UnionUsers
+from users.models import User
+
+
+class VoteSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Union
+        fields = "__all__"
```

Comment:
Ik zou deze initieel zetten op de [post, comment, user, vote]. Als je de model gaat uitbreiden zet je niet perongeluk nieuwe info in je api response. 

Is een comment, mag je zelf wete.